src/Autosuggest.js

```diff
@@ -235,15 +252,19 @@ class Autosuggest extends Component {
           case 'Enter': {
             const focusedSuggestion = this.getFocusedSuggestion();
 
-            if (focusedSuggestion !== null) {
+            if (focusedSuggestion) {
```

Comment:
@moroshko I added a test for this. If you keep this as `!= null` this test till fail: 

https://github.com/moroshko/react-autosuggest/pull/165/commits/36bb114effa5d2ef6a4b1ed96f018138300c1440#diff-d6150ad17107b5367bf67fd9cae74c09R257